build-logic/convention/src/main/kotlin/com/hedvig/android/KotlinAndroid.kt

```diff
@@ -57,10 +57,12 @@ private fun KotlinJvmOptions.configureKotlinOptions(
   allWarningsAsErrors = project.properties["warningsAsErrors"] as? Boolean ?: false
 
   freeCompilerArgs = freeCompilerArgs + listOf(
+    "-opt-in=androidx.compose.material3.ExperimentalMaterial3Api",
+    "-opt-in=androidx.compose.material3.windowsizeclass.ExperimentalMaterial3WindowSizeClassApi",
```

Comment:
I had forgotten we can just just opt-in globally like this. It's imo much preferred over having to have @OptIn(ExperimentalMaterial3Api) in like tens of places all over the codebase.